### ISSUE TITLE
Yet another syntax coloring bug with new f-strings

### ISSUE THREAD

**Issue by edreamleo** [leo-editor/leo-editor#3739]
`test_g_findAnyUnl` contains the following:
```python
gnx_file_head = 'unl:gnx://' + file_name + '#'
```
The following should be equivalent, but Leo does not terminate the f-string properly:
```python
gnx_file_head = f"unl:gnx://{file_name}#"
# oops considered part of f-string!
```

**Comment by tbpassin** [comment]
I'm not seeing what is wrong here.  What is erroneously considered part of the fstring?

**Comment by edreamleo** [comment]
The comment `# oops considered part of f-string!` is colored as part of the f-string.

But only on Python 3.12. Don't worry, I'm on this.

**Comment by edreamleo** [comment]
Closed via PR #3740.
